src/onegov/core/utils.py

```diff
@@ -416,7 +417,16 @@ def linkify(text: str, escape: bool = True) -> str:
     if not text:
         return text
 
-    linkified = linkify_phone(bleach.linkify(text, parse_email=True))
+    long_top_level_domains = ['.agency']
+
+    # bleach.linkify supports only a fairly limited amount of tlds
+    if any(domain in text for domain in long_top_level_domains):
+        if '@' in text:
+            linkified = str(Markup(f'<a href="mailto:{text}">{text}</a>'))
+        else:
+            linkified = str(Markup(f'<a href="{text}">{text}</a>'))
+    else:
+        linkified = linkify_phone(bleach.linkify(text, parse_email=True))
 
```

Comment:
The current TLD list at [main/bleach/linkifier.py#L11-L23](https://github.com/mozilla/bleach/blob/main/bleach/linkifier.py?rgh-link-date=2022-04-22T08%3A05%3A19Z#L11-L23) does not contain a lot of currently-existing TLDs.

We'll have to replace bleach with something (Markupsafe?) anyway (deprecated since January 2023)

Reply:
`bleach` and `Markupsafe` accomplish different goals. `bleach` is still useful for sanitizing user submitted Markup, i.e. in a field where they are allowed to enter HTML, but aren't allowed to insert let's say a script tag with malicious javascript code.

Markupsafe just completely escapes Markup, so it assumes the user input was never meant to be interpreted as markup.

Reply:
It seems easy enough to implement our own linkify, we already did it for phone numbers anyways. So I don't mind stop using the bleach method for urls.